oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -193,7 +180,6 @@ protected GoogleCredentials() {
    * @deprecated Use {@link #create(AccessToken)} instead. This constructor will either be deleted
    *             or made protected/private in a later version.
    **/
-  @Deprecated
```

Comment:
I see `@Deprecated` was removed here.  If it's actively used, please also remove the `@deprecated` comment`

Reply:
done